docs/topics/proc_installing-launcher-operator.adoc

```diff
@@ -0,0 +1,112 @@
+// Redefines the value of the URL placeholder from minishift-specific to a more general value.
+:value-url-osl-auth: OPENSHIFT_CLUSTER_URL
+
+[id='installing-launcher-tool-using-an-operator_{context}']
+= Installing the {name-launcher} tool using an operator
+
+Set up an operator to install the {name-launcher} tool on an OpenShift cluster.
+The following procedure works with clusters running on OpenShift 3.11.
+
+.Prerequisites
+
+* Admin access to an OpenShift cluster.
+z
+.Procedure
+
+. Assign the `cluster-admin` role to your user account on the OpenShift cluster.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc adm policy --as system:admin add-cluster-role-to-user cluster-admin USER_ACCOUNT_NAME
+----
+
+. Log in to your {OpenShiftCluster} using your credentials:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login {value-url-osl-auth}
+----
+
+. Create a project to host the operator:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc new-project launcher-operator
+----
+
+. link:https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line[Create a personal GitHub access token].
+Ensure that you set the following scope when creating the token:
+// TODO: replace this with OAuth: https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps/
+
+* `repo`
+* `admin:repo_hook`
+* `delete_repo`
+
+. Create a new secret using your newly created personal GitHub access token:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc create secret generic launcher-secrets --from-literal=github-token=GITHUB_TOKEN
+----
+
+. Clone the link:https://github.com/fabric8-launcher/launcher-operator[repository] containing the operator and navigate to the directory containing the cloned resources.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ git clone git@github.com:fabric8-launcher/launcher-operator.git
+$ cd launcher-operator
+----
+
+. Process the YAML templates in the `/deploy` subdirectory:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$  oc create -R -f ./deploy
+----
++
+Based on the templates, OpenShift automatically creates:
+
+* a custom resource definition for the Launcher operator,
+* a service account for the Launcher operator,
+* access role definitions for the launcher operator, and
+* access role bindings for the Launcher operator.
+
+. Create the custom resource for the Launcher application.
```

Comment:
I would suggest to say `Install the following custom resource in order to initiate the deployment of the launcher. WDYT ? @inoxx03 

It should be great that Launcher's team document the Launcher Spec API - https://github.com/fabric8-launcher/launcher-operator/blob/master/pkg/apis/launcher/v1alpha2/launcher_types.go and that you add a remark here in order to explain the different parameters that we can customize within the CR and that they also release the operator

Reply:
@cmoulliard Thank you. I have rephrased the instructions based on your suggestion.
I have also discussed changing the contents of the `deploy` directory to a resource list, but was unable to get the changes into the launcher-operator soon enough. Therofe I have replaced the clone step with a ZIP download step and filed a follow-up JIRA to create a resource list and subsequently update the instructions.  I am also filing a JIRA to document the Launcher spec API, which I will do post-summit:)